velox/functions/prestosql/BinaryFunctions.h

```diff
@@ -436,4 +436,107 @@ struct ToIEEE754Bits32 {
   }
 };
 
+template <bool lpad, bool isAscii, typename TOut, typename TIn>
+FOLLY_ALWAYS_INLINE void
+pad(TOut& output, const TIn& input, const int64_t size, const TIn& padding) {
+  static constexpr size_t padMaxSize = 1024 * 1024; // 1MB
```

Comment:
Start constant with 'k' and start variable with min, max, num prefixes, so kMaxPadSize is a better name here.